docs/source/admin/traffic_portal/usingtrafficportal.rst

```diff
@@ -603,6 +621,8 @@ User management includes the ability to (where applicable):
 - update an existing user
 - view :term:`Delivery Service`\ s visible to a user
 
+.. Note:: If OAuth is enabled, creating/deleting a user here will update the user's :term:`role` but the user needs to be created/deleted with the OAuth provider as well.
```

Comment:
This could probably be reworded for clarity that a given username must exist in both authentication systems, but the rights of a TrafficOps user are defined by the role assigned to the TrafficOps user.

Reply:
done